rosplan_planning_system/launch/includes/planner_interface.launch

```diff
@@ -11,7 +11,7 @@
 	<arg name="data_path"            default="$(find rosplan_planning_system)/test/pddl/turtlebot/" />
 	<arg name="planner_command"      default="timeout 10 $(find rosplan_planning_system)/common/bin/popf DOMAIN PROBLEM" />
 	<arg name="planner_interface"    default="popf_planner_interface" />
-
+	<arg name="use_ffha"		 default="false" />
```

Comment:
This comment goes for both this line (14) and 32-35, this launch file contains an example on how to run a generic planner interface, and specifically takes popf as use case. For ff and ffha we have a particular situation where the parsing is shared among both, so put simply falls out of the default case. However is nice to have an example launch file to document how to use the ff / ffha planner interface. I propose to move this launch file to [rosplan_demos](https://github.com/kcl-planning/rosplan_demos), we can create a pkg in the (rosplan_demos) root with name: rosplan_planner_interfaces and store examples there. So for this PR we need to undo the changes in this file